src/shared/Router.tsx

```diff
@@ -0,0 +1,15 @@
+import { BrowserRouter, Routes, Route } from 'react-router';
+import { GiftPage } from '@/pages/GiftPage';
+import { Login } from '@/pages/Login';
+import { NotFound } from '@/pages/NotFound';
+export const Router = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<GiftPage />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="*" element={<NotFound />} />
+      </Routes>
```

Comment:
지금은 Router 컴포넌트 안에 라우팅 정보가 모두 하드코딩되어있는데요,
만약 규모가 커지게 되면 이러한 방식으로는 사용하기가 어려울 수 있습니다.

```
// routes.ts
import { GiftPage } from '@/pages/Home'

export const routes = [
  { path: '/', element: <GiftPage /> },
]
```

따라서 라우팅 정보는 위와 같이 별도 상수 파일로 분리하고,

```suggestion
      <Routes>
      {routes.map(({ path, element }) => (
        <Route key={path} path={path} element={element} />
      ))}
    </Routes>
```
위와 같이 해당 데이터를 활용하는 것도 좋은 방법일 것 같습니다.